README.md

```diff
@@ -225,9 +212,9 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
```

Comment:
- [ ] Since this is not integral to your personal content i.e. it is part of a template, I think removing it makes your code even cleaner. I don't think it is information that you want users to respond to especially considering the fact that they are not expected to write anything in your code (no right)! Kindly remove this sectional description to avoid ambiguity. :ok_hand: